tests/plugins/logger/logger_test.go

```diff
@@ -74,6 +76,69 @@ func TestLogger(t *testing.T) {
 	wg.Wait()
 }
 
+func TestLoggerRawMode(t *testing.T) {
+	out := capturer.CaptureOutput(func() {
+		container, err := endure.NewContainer(nil, endure.RetryOnFail(true), endure.SetLogLevel(endure.ErrorLevel))
+		if err != nil {
+			t.Fatal(err)
+		}
+		// config plugin
+		vp := &config.Viper{}
+		vp.Path = "configs/.rr-raw-mode.yaml"
+		vp.Prefix = "rr"
+
+		err = container.RegisterAll(
+			vp,
+			&Plugin{},
```

Comment:
@bricelalu You are testing a little bit different things here. Raw mode is not the mode for the plugin, it's for PHP. 
Simple diagram:
PHP -> `stderr` -> `logger` -> check the whole line. That should be the only string, that you write from the PHP.
To test log output (from go -> go), you can simply use `mock.logger` like here for example https://github.com/spiral/roadrunner/blob/master/tests/plugins/http/http_plugin_test.go#L712
Just a little suggestion:
1. Create (or re-use) a PHP file (there are a bunch of them in the `tests` folder).
2. Write to the stderr from the PHP and get the output in Go. I guess, that you might use `mock` logger for that or grab the output as you did before.

Reply:
And for sure, thank you very much for your work on this PR :)

Reply:
Thank you very much for your guidance :) it really helps :)

I will do that: create a PHP file that writes to `stderr` then grab the output. Thank you very much also, I learned a lot :)